Compilation fails on MacOS and Linux with clang

```
src/interpreter.cpp:657:66: error: cannot pass non-trivial object of type 'std::string' (aka 'basic_string<char>') to variadic function; expected type from format string was 'char *' [-Wnon-pod-varargs]
  657 |         printf("Could not find a symbol in expression \'%s\'\n", expr->print(0));
      |                                                         ~~       ^~~~~~~~~~~~~~
src/interpreter.cpp:657:66: note: did you mean to call the c_str() method?
  657 |         printf("Could not find a symbol in expression \'%s\'\n", expr->print(0));
      |                                                                  ^             
      | 
```
Cool project. I cloned the repo to check it out and got these two errors while trying to compile it with clang, tested on both MacOS and Linux.

--- a/src/interpreter.cpp
+++ b/src/interpreter.cpp
@@ -629,7 +629,7 @@ koala::interpreter::symbol* koala::interpreter::search_symbol(koala::expression*
 
             if (struct_sym->v->get_class() != TP_STRUCT) {
                 printf("Symbol \'%s\' does not refer to a struct type object\n",
-                    struct_sym->name
+                    struct_sym->name.c_str()
                 );
 
                 std::exit(1);
@@ -654,7 +654,7 @@ koala::interpreter::symbol* koala::interpreter::search_symbol(koala::expression*
     }
 
     if (!sym) {
-        printf("Could not find a symbol in expression \'%s\'\n", expr->print(0));
+        printf("Could not find a symbol in expression \'%s\'\n", expr->print(0).c_str());
 
         std::exit(1);
     }

